CapMonsterCloud.Client.Tests/JsonConverterTests.cs

```diff
@@ -0,0 +1,86 @@
+﻿using FluentAssertions;
+using Newtonsoft.Json;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using Zennolab.CapMonsterCloud.Requests;
+
+namespace Zennolab.CapMonsterCloud.Client
+{
+    public class JsonConverterTests
+    {
+        [Test]
+        public void NumericFlagConverter_ShouldDeserialize([Values] bool numeric)
+        {
+            // Arrange
+            var target = new ImageToTextRequest
+            {
+                Body = "some base64 body",
+                CapMonsterModule = CapMonsterModules.YandexWave,
+                CaseSensitive = true,
+                Numeric = numeric,
+                RecognizingThreshold = 65,
+                Math = false
+            };
+
+            // Act
+            var actual = JsonConvert.SerializeObject(target);
```

Comment:
Act section should contain only one operation, which is under test. In these tests I suppose it should be `JsonConvert.DeserializeObject...`

Reply:
I made changes in JsonConverterTests methods to match the AAA pattern, as you asked.